templates/cloudflare-workers/vite.config.ts

```diff
@@ -4,10 +4,13 @@ import {
   cloudflareDevProxyVitePlugin,
 } from "@remix-run/dev";
 import tsconfigPaths from "vite-tsconfig-paths";
+import { getLoadContext } from "./load-context";
 
 export default defineConfig({
   plugins: [
-    cloudflareDevProxyVitePlugin(),
+    cloudflareDevProxyVitePlugin({
+      getLoadContext,
+    }),
     remix({
       future: {
```

Comment:
@edmundhung let's go ahead and add 

```
v3_lazyRouteDiscovery: true,
v3_singleFetch: true,
```

to the config

Reply:
I see that you added the config already. I have rebased my branch to reflect those.